metriq-api/models/userModel.js

```diff
@@ -20,6 +20,10 @@ module.exports = function (sequelize, DataTypes) {
       type: DataTypes.TEXT,
       allowNull: false
     },
+    affiliation: {
+      type: DataTypes.TEXT,
+      allowNull: true
```

Comment:
Vincent, right here:
```
allowNull: false,
defaultValue: ''
```

Reply:
Ah, okay. I guess I wasn't aware that it needed an explicit default value (maybe I thought that since the datatype was text and it could be null that it would imply this would be the empty string). Also, wasn't aware of the `defaultValue` param, so thanks for that. Added.

Reply:
By the way, with the default value, it shouldn't be necessary to change your other code, hopefully: `null` values in this model field will be populated with the default value instead.

Reply:
Ah okay, so even though the value *could* be left blank from the user we don't necessarily treat this as a null value. Okay, I guess my thought process was that we wouldn't enforce the user to put this info in and it could therefore be null, but I think what you're proposing makes more sense.

Reply:
Gotcha. Should we do `allowNull: false` or `allowNull: true`. Any prefernece? Right now it is `false`, but I can see an argument for both.

Reply:
`null` values make a mess in the database, whenever they can be reasonably avoided. In the case of strings, in general, empty string can be set to default, so we basically never need nullable strings.